spec/dealership_spec.rb

```diff
@@ -0,0 +1,145 @@
+require 'rspec'
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+
+  it 'inventory' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory).to eq([])
+  end
+
+  it 'inventory count' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory_count).to eq(0)
+  end
+
+  it 'inventory - add two cars' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+
+    expect(dealership.inventory).to eq([car_1, car_2])
+  end
+
+  it 'inventory count - add two cars' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+
+    expect(dealership.inventory_count).to eq(2)
+  end
+
+  it 'has_inventory? - false' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.has_inventory?).to eq(false)
+  end
+
+  # might not need this one? interaction pattern doesn't specify it, but thought
+    # it would be a good check?
+  it 'has_inventory? - true' do
```

Comment:
I think it's good to check both the true and false paths! I might name them `has_inventory? returns false with no cars` and `has_inventory? returns true when there are cars`